bundles/FileExplorerBundle/src/Controller/Admin/AdminController.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Pimcore\Bundle\FileExplorerBundle\Controller\Admin;
+
+use Pimcore\File;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/file-explorer")
+ *
+ * @internal
+ */
+class AdminController extends \Pimcore\Bundle\AdminBundle\Controller\AdminController
```

Comment:
I am not sure if we should put all in one controller. Maybe we should keep the original structure a little more and move it to a MiscController instead? At least that is what I did e.g. for the glossary. Wdyt?

Reply:
The methods inside the AdminController serve a single purpose of handling the actions for creating/updating/deleting the files inside the file-explorer tree. I think it makes more sense to keep them together and not split into multiple places as debugging in future can be painful because of that